lib/chef/azure/service.rb

```diff
@@ -155,4 +116,149 @@ def is_running?
     end
     return false
   end
+
+  def start_service
+    result = shell_out("sc.exe start chef-client")
+    result.error!
+  end
+
+  def set_interval(client_rb, new_chef_service_interval)
+    client_rb_contents = read_client_rb(client_rb)
+
+    if interval_exist?(client_rb_contents)
+      interval = interval_index(client_rb_contents)
+      client_rb_contents[interval] = interval_string(new_chef_service_interval)
+    else
+      client_rb_contents << interval_string(new_chef_service_interval)
+    end
+
+    write_client_rb(client_rb, client_rb_contents.join)
+  end
+
+  def disable_service

```

Comment:
lets rename this to stop_service since stoping and disabling a service are distinctly different